release-notes/release-notes507.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: default
+title: TinyMCE 5.0.7
+title_nav: TinyMCE 5.0.7
+keywords: releasenotes newfeatures deleted technologypreview bugfixes knownissues
+---
+
+## Overview
+
+The Release Notes provide high-level coverage of the improvements and additions that have been implemented in TinyMCE 5.0.7 and document known problems in this release, as well as important bug fixes, deprecated functionality, and other details.
+
+TinyMCE 5.0.7 release adds improvements to the TinyMCE editor to improve the overall user experience.
+
+## New features
+
+### TinyMCE
+
+#### Help API
+
+The **Help** plugin includes a new `help_tabs` setting and an API to be able to add tabs from within custom plugins. This allows for the default tabs to be enabled/disabled, change the tab order or to add additional tabs to the help dialog.
+
+For more information on the new `help_tabs` setting and API refer to the [documentation]({{site.baseurl}}/plugins/help/).
+
+#### Image Tools custom image fetch
+
+The **Image Tools** plugin includes a new `imagetools_fetch_image` setting, which allows a custom function to be used to fetch the image content for more complex, such as when authentication is required.
```

Comment:
the image content for more complex, such as when authentication is required.

the image content in complexed environments where extra customisation is required such as authentication

Reply:
Updated, though I did some commas and changed `complexed` to `complex`.

Reply:
and oops, it was meant to say "complex situations" originally sorry!